src/server_manager/web_app/i18n_formatting.ts

```diff
@@ -0,0 +1,28 @@
+/*
+  Copyright 2020 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// Utility functions for internationalizing units and currency
+
+export function makeUnitFormatter(unit: string, language: string) {
```

Comment:
Let's replace this with our own version of the of byte_size instead.

Also, code that is not application-specific should live in `infrastructure/`

Reply:
Done in 79fb1ea